test/import/FSHImporter.Invariant.test.ts

```diff
@@ -0,0 +1,126 @@
+import { importSingleText } from '../testhelpers/importSingleText';
+import { FshCode } from '../../src/fshtypes';
+import { loggerSpy } from '../testhelpers/loggerSpy';
+
+describe('FSHImporter', () => {
+  describe('Invariant', () => {
+    describe('#invariantMetadata', () => {
+      it('should parse the simplest possible invariant', () => {
```

Comment:
I think this is actually technically invalid. It looks like on `constraint`, `key`, `severity` and `human` are all required, which corresponds to an Invariant that must have a `Description` and a `Severity`. So if someone is making an invalid Invariant, we should note that somewhere, either in the importer or the exporter. I don't know necessarily which makes more sense, do you have thoughts on that?

Reply:
I think it would be reasonable to check for that and log an error in `visitInvariant`, since I don't think it is possible for additional metadata to be added once the invariant parser rule is finished.